great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -341,9 +341,8 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
         self._anonymizer = Anonymizer(salt=self._data_context_id)
 
         # TODO: Need to determine V1 mechanism for usage stats opt-out
```

Comment:
I think you can remove this comment now. We can file a follow up ticket for making this config in addition to the env variable control.